frontend/src/components/CaseContainer.js

```diff
@@ -347,6 +370,71 @@ class CaseContainer extends Component {
     );
   }
 
+  enableEditing() {
+    if (!this.state.assurance_case.lock_uuid) {
+      this.submitCaseChange("lock_uuid", this.state.session_id);
+    } else if (this.state.assurance_case.lock_uuid !== this.state.session_id) {
+      // override!
+      if (
+        window.confirm(
+          "Are you sure?  You might be overwriting someone's work..."
+        )
+      ) {
+        this.submitCaseChange("lock_uuid", this.state.session_id);
+      }
+    }
+    this.updateView();
```

Comment:
I wonder if we might hit race conditions, where the browser tells the backend to set `lock_uuid` and then immediately reloads, but the backend hasn't yet had time to process the request, and sends the old value? Same question for `disableEditing`.

Reply:
Yep, good point!   I hadn't seen this race condition in action, but it's good to be as robust as we can...  
I had a go at fixing this by putting the `updateView` inside a `then( ... )` - let me know if you think this is the right way to do it?

Reply:
Looks good to me